ssclj/rpm/src/main/systemd/ssclj.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
 Description=SlipStream API
-After=syslog.target network.target
-After=hsqldb.service elasticsearch.service
+After=syslog.target
+After=network.target
+After=hsqldb.service
```

Comment:
No, hsqldb is still needed because of the authentication.  The checking/adding of users is done directly in the hsqldb rather than through the SlipStream interface.  Once the user resource moves to clojure, this dependency can be removed.